src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+function RegistrationForm(props) {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+  const handleChange = (e) => {
+    const { id, value } = e.target;
+    setState((prevState) => ({
+      ...prevState,
+      [id]: value,
+    }));
+  };
+  const handleSubmitClick = (e) => {
+    e.preventDefault();
+    if (state.password === state.confirmPassword) {
+      console.log("email: ", state.email, "password: ", state.password);
+    } else {
+      console.log("Passwords do not match!");
+    }
+  };
+  return (
+    <div className="card col-12 col-lg-4 login-card mt-2 hv-center">
```

Comment:
What does these classes do?

Reply:
These are bootstrap classes used for inputs and the button.